addons/website_blog/static/src/js/website_blog.js

```diff
@@ -114,26 +97,22 @@ publicWidget.registry.websiteBlog = publicWidget.Widget.extend({
         }
         window.open(url, '', 'menubar=no, width=500, height=400');
     },
+
+    //--------------------------------------------------------------------------
+    // Utils
+    //--------------------------------------------------------------------------
+
     /**
      * @private
-     * @param {Event} ev
-     */
-    _onYearCollapseClick: function (ev) {
-        this._toggleYearCollapse($(ev.currentTarget));
-    },
-    /**
-     * @private
-     * @param {Event} ev
-     */
-    _onBlogPostMouseEnter: function (ev) {
-        $('div.o_blog_post_complete a').not('#' + ev.srcElement.id).addClass('unhover');
-    },
-    /**
-     * @private
-     * @param {Event} ev
+     * @param {Boolean} compensateAffix if set to true, double the nav height
+     *                  value to compensate the affix effect
      */
-    _onBlogPostMouseLeave: function (ev) {
-        $('div.o_blog_post_complete a').not('#' + ev.currentTarget.id).removeClass('unhover');
+    _computeNavHeight: function (compensateAffix) {
+        var $mainNav = $('#wrapwrap > header > nav');
```

Comment:
Should not it be the `header` itself ?

Reply:
Indeed! Actually, you spotted a bigger issue, I was targeting the wrong element ( the post content rather than the post content's section).
This is why `> header > nav` worked, it was just a coincidence!
thanks!

Reply:
@qsm-odoo , I changed this part with ac64a5b697ce47b94d21d279608d970563841ed3.
Since you exposed the `nav` issue I was able to simplify a lot, removing the `compensateAffix` and using the same animation for both `_onNextBlogClick` and `_onContentAnchorClick`.

You should check it again, sorry for the hassle.